<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
    <line>
      <![CDATA[chrome launched]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="45360" started-at="2019-05-08T11:47:32Z" finished-at="2019-05-08T11:48:17Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="45360" started-at="2019-05-08T11:47:32Z" finished-at="2019-05-08T11:48:17Z">
      <class name="com.quadlabs.pagescript.hotel.LoginHotelTest">
        <test-method status="PASS" signature="precondition(java.lang.String)[pri:0, instance:com.quadlabs.pagescript.hotel.LoginHotelTest@4501b7af]" name="precondition" is-config="true" duration-ms="4517" started-at="2019-05-08T17:17:32Z" finished-at="2019-05-08T17:17:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[chrome launched]]>
            </line>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="validlogintest()[pri:0, instance:com.quadlabs.pagescript.hotel.LoginHotelTest@4501b7af]" name="validlogintest" duration-ms="40831" started-at="2019-05-08T17:17:36Z" finished-at="2019-05-08T17:18:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validlogintest -->
        <test-method status="FAIL" signature="postcondition(org.testng.ITestResult)[pri:0, instance:com.quadlabs.pagescript.hotel.LoginHotelTest@4501b7af]" name="postcondition" is-config="true" duration-ms="0" started-at="2019-05-08T17:18:17Z" finished-at="2019-05-08T17:18:17Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[wrong number of arguments]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:232)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postcondition -->
      </class> <!-- com.quadlabs.pagescript.hotel.LoginHotelTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
